cmd/frontend/graphqlbackend/search.go

```diff
@@ -505,9 +501,54 @@ func resolveRepositories(ctx context.Context, op resolveRepoOp) (repoRevisions,
 	}
 	tr.LazyPrintf("Associate/validate revs - done")
 
+	if op.commitAfter != "" {
+		repoRevisions = filterRepoHasCommitAfter(ctx, repoRevisions, op.commitAfter)
+	}
+
 	return repoRevisions, missingRepoRevisions, repoResolvers, overLimit, nil
 }
 
+func filterRepoHasCommitAfter(ctx context.Context, revisions []*search.RepositoryRevisions, after string) []*search.RepositoryRevisions {
+	var (
+		mut  sync.Mutex
+		pass = []*search.RepositoryRevisions{}
+		res  = make(chan *search.RepositoryRevisions, 100)
+		run  = parallel.NewRun(1000)
```

Comment:
```suggestion
		run  = parallel.NewRun(256)
```

I forgot to push this change. I think we need to use something lower like this or risk flooding gitserver too much with requests. Better to start low and we can adjust later if needed (likely with batching).